src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -62,6 +62,27 @@ paths: {
 }
 ```
 
+Consider the example of overwriting an HTML file in the adminhtml.
+In this example, the `max-length` value of the text-box in the `adminhtml` is altered. The HTML file is located at `vendor/magento/module_ui/view/base/web/templates/form/element/input.html`
+
+1. Create a `requirejs-config.js` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/`
+
+    ```javascript
+    var config = {
+        paths: {
+            'ui/template/form/element/input': '<vendor_name>_<module_name>/template/form/element/input'
+        }
+    };
+    ```
+
+1. Create an `input.html` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/web/template/form/` and copy the contents of the `input.html` file from the `module_ui` template file.
+1. Change the maxlength value to be `512` which was originally set to be `256`.
+1. Run the Magento setup upgrade and setup compile commands.
```

Comment:
Convert this step into two:

1. Upgrade the Magento application:

   ```bash
   bin/magento setup:upgrade
   ```

1. Generate the dependency injection configuration:

   ```bash
   bin/magento setup:di:compile
   ```